tests/unit/components/ValueTypeDropDown.spec.ts

```diff
@@ -8,15 +8,10 @@ describe( 'ValueTypeDropDown.vue', () => {
 		const optionItems = PropertyValueRelation;
 		const wrapper = shallowMount( ValueTypeDropDown );
 
-		wrapper.setData( {
-			selected: optionItems.Matching,
-			optionItems: optionItems,
-		} );
-
-		wrapper.findComponent( ValueTypeDropDown ).vm.$emit( 'input', optionItems.Matching );
+		wrapper.find(`option[value="${optionItems.Regardless}"]`).setSelected();
```

Comment:
It is failing on the linter. Other than that looks good. :+1:

Reply:
Thanks, fixed it!